universe/Building.cpp

```diff
@@ -228,34 +228,34 @@ std::string BuildingType::Dump() const {
     }
 
     if (m_location) {
-        retval += DumpIndent() + "location = \n";
-        ++g_indent;
-            retval += m_location->Dump();
-        --g_indent;
+        retval += DumpIndent(ntabs) + "location = \n";
+        ++ntabs;
+            retval += m_location->Dump(ntabs);
+        --ntabs;
```

Comment:
These simple instances of increment, pass, decrement could now be reduced to `->Dump(ntabs + 1)`

Reply:
Good point.  I made the changes.